nltk/tag/hunpos.py

```diff
@@ -106,14 +109,14 @@ def tag(self, tokens):
             assert "\n" not in token, "Tokens should not contain newlines"
             if isinstance(token, compat.text_type):
                 token = token.encode(self._encoding)
-            self._hunpos.stdin.write(token + "\n")
+            self._hunpos.stdin.write(token + self._endl)
```

Comment:
I think `b"\n"` is enough. Every sane encoding will encode `"\n"` to `b"\n"`. The same applies to `"\t"`.